### PR TITLE
[4.0] accessible validation

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
Joomla 4 introduced inline validation messages per field. Unfortunately the implementation was no accessible. This PR fixes that. Also Improves accessibility to the administration login page

- Inline validation messages need to be part of the label

### Testing Instructions
Apply PR, Logout try to login in administration area, without filling the username and password, or try to tab through the fields.

Create a new article and try to save it without filling a title.

The error message should be **focused**. If you have a mac turn on VoiceOver and redo the previous step: you should hear the error message!

### Preview
screen shot 2017-07-28 at 17 48 54
screen shot 2017-07-28 at 17 48 26

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -40,28 +41,30 @@
 	');
 }
 ?>
-<form class="login-initial" action="<?php echo JRoute::_('index.php', true); ?>" method="post" id="form-login">
+<form class="login-initial form-validate" action="<?php echo JRoute::_('index.php', true, $params->get('usesecure')); ?>" method="post" id="form-login">
```
Comment (wilsonge) [inline administrator/modules/mod_login/tmpl/default.php]: `usesecure` param was deliberately removed

File: administrator/modules/mod_login/tmpl/default.php [hunk 2]
```diff
@@ -40,28 +41,30 @@
...
 
 		<div class="form-group">
 			<label for="mod-login-username"><?php echo JText::_('JGLOBAL_USERNAME'); ?></label>
 			<input
-				name="username"
-				id="mod-login-username"
-				type="text"
-				class="form-control input-full"
-				autofocus
+					name="username"
```
Comment (C-Lodder) [inline administrator/modules/mod_login/tmpl/default.php]: remove 1 tab from these lines

File: administrator/modules/mod_login/tmpl/default.php [hunk 3]
```diff
@@ -40,28 +41,30 @@
...
 
 		<div class="form-group">
 			<label for="mod-login-password"><?php echo JText::_('JGLOBAL_PASSWORD'); ?></label>
 			<div class="input-group">
 				<input
-					name="passwd"
-					id="mod-login-password"
-					type="password"
-					class="form-control input-full"
+						name="passwd"
```
Comment (C-Lodder) [inline administrator/modules/mod_login/tmpl/default.php]: and these lines

File: administrator/modules/mod_login/tmpl/default.php [hunk 4]
```diff
@@ -74,11 +77,12 @@ class="form-control input-full"
 			<label for="mod-login-secretkey"><?php echo JText::_('JGLOBAL_SECRETKEY'); ?></label>
 			<div class="form-group">
 				<input
-					name="secretkey"
-					autocomplete="off"
-					id="mod-login-secretkey"
-					type="text"
-					class="form-control input-full"
+						name="secretkey"
```
Comment (C-Lodder) [inline administrator/modules/mod_login/tmpl/default.php]: and these lines

File: administrator/templates/atum/login.php [hunk 5]
```diff
@@ -67,6 +67,9 @@
 			display: none;
 		}
 		<?php endif; ?>
+		 .form-control-feedback {
```
Comment (C-Lodder) [inline administrator/templates/atum/login.php]: You've added this to the SCSS file (https://github.com/joomla/joomla-cms/pull/17318/files#diff-04ab4085c79f8cbbfd51919364387f7bR113) so no need to add it here.

**Pull request merged**